### PR TITLE
Update the docs

### PR DESCRIPTION
**Pull request by sz-piotr**

### CODE REVIEW

File: docs/source/getting-started.rst [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        wallet, // a wallet to sign transactions
+        Counter, // the compiled output
+        [initialValue], // arguments to the smart contract constructor
+      );
+      return counter; // an ethers 'Contract' class instance
+    }
+
+    it('sets initial value in the constructor', async () => {
+      const counter = await deployCounter(200);
+      expect(await counter.value()).to.eq(200);
```
Comment (Anze1m) [inline docs/source/getting-started.rst]:
```suggestion
      expect(await counter.value()).to.equal(200);
```

File: docs/source/getting-started.rst [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    });
+
+    it('can increment the value', async () => {
+      const counter = await deployCounter(200);
+      await counter.increment(42);
+      expect(await counter.value()).to.eq(242);
```
Comment (Anze1m) [inline docs/source/getting-started.rst]:
```suggestion
      expect(await counter.value()).to.equal(242);
```

File: docs/source/getting-started.rst [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+Running tests
+-------------
+
+In order to run tests you will need to install a test runner. We recomment using
+`Mocha <https://mochajs.org/>`__ alongide with `Chai <https://www.chaijs.com/>`__.
+
+Run:
+::
+
+  yarn add --dev mocha chai
```
Comment (Anze1m) [inline docs/source/getting-started.rst]: Shouldn't we have `mocha` and `chai` installed before writing tests?

File: docs/source/matchers.rst [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+A set of sweet chai matchers, makes your test easy to write and read. Below is the list of available matchers:
+
+Bignumbers
+----------
+Testing equality of big numbers:
+
+::
+
+  expect(await token.balanceOf(wallet.address)).to.eq(993);
```
Comment (Anze1m) [inline docs/source/matchers.rst]:
```suggestion
  expect(await token.balanceOf(wallet.address)).to.equal(993);
```

File: docs/source/fixtures.rst [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    async function fixture(provider, [wallet, other]) {
+      const token = await deployContract(wallet, BasicTokenMock, [
+        wallet.address, 1000
+      ]);
+      return {token, wallet, other};
+    }
+
+    it('Assigns initial balance', async () => {
+      const {token, wallet} = await loadFixture(fixture);
+      expect(await token.balanceOf(wallet.address)).to.eq(1000);
```
Comment (Anze1m) [inline docs/source/fixtures.rst]:
```suggestion
      expect(await token.balanceOf(wallet.address)).to.equal(1000);
```

File: docs/source/fixtures.rst [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    });
+
+    it('Transfer adds amount to destination account', async () => {
+      const {token, other} = await loadFixture(fixture);
+      await token.transfer(other.address, 7);
+      expect(await token.balanceOf(other.address)).to.eq(7);
```
Comment (Anze1m) [inline docs/source/fixtures.rst]:
```suggestion
      expect(await token.balanceOf(other.address)).to.equal(7);
```

**Comment by Anze1m** [comment]
In some examples, this formatting is used `{something}`, in some that `{ something }`. We should pick one and keep consistent.

File: docs/source/getting-started.rst (hunk 3 above)
Comment (sz-piotr) [reply docs/source/getting-started.rst]: Not if you are writing them in javascript (as in the example)

**Pull request merged**